### ISSUE TITLE
How to add content scripts to dist without using manifest.json

### ISSUE THREAD

**Issue by pyoner** [NekitCorp/chrome-extension-svelte-typescript-boilerplate#26]
I want to use [chrome.scripting.executeScript](https://developer.chrome.com/docs/extensions/reference/api/scripting#method-executeScript) to execute content scripts.

How to add path of content scripts?

**Comment by NekitCorp** [comment]
Can this help you? 

https://github.com/NekitCorp/chrome-extension-svelte-typescript-boilerplate/discussions/24

**Comment by pyoner** [comment]
> Can this help you?
> 
> #24

@NekitCorp thanks, it works for me.

```typescript
import injectProgrammatically from "../content/inject-programmatically?script";
```
what Vite plugin does this?

**Comment by NekitCorp** [comment]
> > Can this help you?
> > #24
> 
> @NekitCorp thanks, it works for me.
> 
> ```ts
> import injectProgrammatically from "../content/inject-programmatically?script";
> ```
> 
> what Vite plugin does this?

Yes, you can read more about it here: https://dev.to/jacksteamdev/advanced-config-for-rpce-3966#dynamic-content-scripts

**Comment by pyoner** [comment]
@NekitCorp thanks a lot!
